tgui/packages/tgui/format.js

```diff
@@ -111,6 +111,30 @@ export const formatDb = value => {
   return sign + formatted + ' dB';
 };
 
+/**
+ * Formats time as a string in the minutes:seconds format.
+ * @param time the time to format, in tenths of a second
+ * @param msg an optional message to display if time <= 0
+ * @example formatTime(690)
+ * //returns `01:09`
+ * @example formatTime(0, 'BO:OM')
+ * //returns `BO:OM`
+ */
+export const formatTime = (time, msg = "") => {
```

Comment:
Went with 1/10 of a second to leave room for potentially expanding this, maybe something like being able to produce a number of different formats based on some flag argument, idk